Assets/Scripts/NPC/CeilingDetection/CeilingRayCastDetection.cs

```diff
@@ -0,0 +1,28 @@
+using UnityEngine;
+
+public class CeilingRayCastDetection : MonoBehaviour
+{
+    [SerializeField] private Transform raycastOrigin;
+    [SerializeField] private float raycastLength = 2f;
+    [SerializeField] private LayerMask ceilingLayer;
+    [SerializeField] private Animator animator;
+
+    private bool isTouchingCeiling = false;
```

Comment:
```suggestion
    private bool isTouchingCeiling;
```
Default value voor een bool is altijd false.